style.css

```diff
@@ -515,3 +948,479 @@ body {
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+}
+
+@media (min-width: 520px) {
+  .bottom-section__text-block {
+    margin-left: 20%;
+    margin-right: 10%;
+  }
+
+  .bottom-section__image {
+    padding-left: 35px;
+    padding-right: 30px;
+  }
+}
+
+@media (min-width: 768px) {
+  .bottom-section__block {
+    margin-left: 30%;
+    margin-right: 15%;
+  }
+
+  .bottom-section__image {
+    padding-left: 45px;
+    padding-right: 40px;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__block {
+    position: absolute;
+    top: 30%;
+    right: 3%;
+    padding: 30px 40px;
+    margin-left: 50%;
+    margin-right: 0;
+  }
+
+  .bottom-section__image {
+    padding:0;
+    margin-right: 40%;
+  }
+
+  .bottom-section__img {
+    margin: 137px 0;
+  }
+
+  .bottom-section__text-block {
+    -webkit-box-align: start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__block {
+    margin-right: 7%;
+  }
+
+  .bottom-section__image {
+    margin-left: 3%;
+  }
+}
+
+@media (min-width: 380px) {
+  .bottom-section__btn_other {
+    right: -57%;
+  }
+}
+
+@media (min-width: 620px) {
+  .bottom-section__btn_other {
+    right: -52%;
+    width: 254px;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__title,
+  .bottom-section__subtitle,
+  .bottom-section__description {
+    margin-left: -80px;
+  }
+
+  .bottom-section__btn_other {
+    position: absolute;
+    right: 8%;
+    bottom: 11%;
+    padding: 0;
+    width: 345px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 13px;
+    line-height: normal;
+  }
+
+  .bottom-section__btn_basket {
+    margin-top: -12.5%;
+    margin-left: 30%;
+    height: 43px;
+    text-align: center;
+  }
+
+  .bottom-section__arrow div {
+    bottom: 7px;
+    left: 240px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__btn_other {
+    right: 10%;
+    bottom: 21%;
+  }
+
+  .bottom-section__btn_basket {
+    margin-top: -12.5%;
+    margin-left: 30%;
+  }
+
+  .bottom-section__arrow div {
+    bottom: 7px;
+    left: 240px;
+  }
+}
+
+@media (max-width: 768px) {
+  .bottom-section__btn_basket {
+    display: flex;
+    justify-content: center;
+  }
+
+  .bottom-section__arrow div {
+    top: 15px;
+  }
+
+  .bottom-section__btn_other {
+    right: -50%;
+  }
+}
+
+@media (min-width: 1200px) {
+  .footer {
+    display: block;
+    position: relative;
+    top: -6%;
+  }
+
+  .footer__whs {
+    position: absolute;
+    right: 75px;
+    bottom: 20px;
+  }
+
+  .footer__developed {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 12px;
+    line-height: normal;
+    text-transform: uppercase;
+  }
+
+  .footer__whs-logo {
+    position: absolute;
+    top: -2px;
+    left: 127px;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)